core/evancore.vue.libs/src/components/dapp-wrapper/dapp-wrapper.ts

```diff
@@ -373,6 +380,10 @@ export default class DAppWrapperComponent extends mixins(EvanComponent) {
     if (this.dispatcherHandler) {
       this.dispatcherHandler.destroy();
     }
+    // stop the ui session and runtime updates
+    if (this.stopSessionWatcher) {
```

Comment:
better check whether it is really a function?

```suggestion
    if (typeof this.stopSessionWatcher === 'function') {
```

Reply:
What else should it be? :D